Replace Lorem Ipsum with Actual Testimonial Data

SanjeevDeori opened the pull request:
Replaced placeholder "Lorem Ipsum" text in the Testimonials section with real testimonial data as per Issue #16 

✅ Changes Made:
Updated testimonial content with real user messages.

Ensured formatting and styling remain consistent.

Fixed display issue where testimonials showed default text.

📌 Related Issue:
Closes #16 

### Before
<img width="903" height="542" alt="Screenshot 2025-07-28 123939" src="https://github.com/user-attachments/assets/378c7d09-99b6-44a4-8888-2904f9f8eb17" />

### After
<img width="905" height="536" alt="Screenshot 2025-07-28 123815" src="https://github.com/user-attachments/assets/7720e88e-2ece-4253-8859-018ee2ae6554" />

vishanurag reviewed (changes requested):
<img width="469" height="465" alt="image" src="https://github.com/user-attachments/assets/07935f17-81b9-48e2-9341-5b56531a2020" />
<br>
please make sure that there is no Mr Anand Singh over here 😅
Please make this testimoniol name as Mr. PQR only. I request

vishanurag reviewed (approved):
LGTM

SanjeevDeori commented:
As requested, I have updated the testimonial name to "Mr. PQR". Please let me know if any further changes are needed. 😊


> <img alt="image" width="469" height="465" src="https://private-user-images.githubusercontent.com/126074487/471409406-07935f17-81b9-48e2-9341-5b56531a2020.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM2OTM1NzQsIm5iZiI6MTc1MzY5MzI3NCwicGF0aCI6Ii8xMjYwNzQ0ODcvNDcxNDA5NDA2LTA3OTM1ZjE3LTgxYjktNDhlMi05MzQxLTViNTY1MzFhMjAyMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyOFQwOTAxMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZDQ3MmExMTk5ZTgwYzBmZDY5NjM5MzkwNmU4Y2I4NmI4NTAwZTZmNWJiZWNjNmQyNzEwZGM0NmY4NjIyNDVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.HOLFhBwByYxjy8JVtFTx78L6yV7ePEnvaq90-TGyqe8">
> please make sure that there is no Mr Anand Singh over here 😅 Please make this testimoniol name as Mr. PQR only. I request

As requested, I have updated the testimonial name to "Mr. PQR". Please let me know if any further changes are needed. 😊
